### ISSUE TITLE
Bar doesn't progress if there are http calls before ngx-loading-bar is loaded

### ISSUE THREAD

**Issue by Spanja** [aitboudad/ngx-loading-bar#20]
Hello,
I get a similar issue as #14  with the new alpha 3 :

If I access my app with the root URL and then navigate to a lazy loaded module, the loading bar is well displayed on each HttpClient call.
But if I then refresh my app being in my lazy loaded module (I press F5), I can't see the bar anymore.
It seems that the bar is well displayed only if I load app with the root URL.

Here is my shared module :

```
export class SharedModule {
  static forRoot() {
    return [
      { ngModule: SharedModule },
      HttpClientModule,
      LoadingBarHttpClientModule,
    ];
  }
}
```

I get same results with declaring LoadingBarHttpClientModule only in app module.

I'm using the new HttpClientModule instead of HttpModule

**Comment by aitboudad** [comment]
@Spanja Hi, can you provide me an example using https://stackblitz.com/ that reproduce your issue, thanks!

**Comment by Spanja** [comment]
Ok The bar is not showing randomly and I figured out why, that's not related to lazy loading.

I created a stackblitz here : https://stackblitz.com/edit/angular-u21ba9?file=app%2Fapp.module.ts

The project includes the ngx-translate module, which retrieves my locale with httpclient before the ngx-loading-bar component is displayed, that's why the loading bar is not displayed with next http calls.
If you comment the appmodule constructor, the bar is displayed correctly when you click on the get button.

Hope this helps.
